src/components/Questionnaire/QuestionTypes/DeathQuestion.tsx

```diff
@@ -25,6 +25,11 @@ export function TimeOfDeathQuestion(props: TimeOfDeathQuestionProps) {
   const values = (questionnaireResponse.values?.[0]?.value as string[]) || [];
 
   const handleUpdate = (updates: string) => {
+    const selectedDate = new Date(updates);
+    const now = new Date();
+    if (selectedDate > now) {
+      return;
```

Comment:
why complicate a lot? just simply use isFuture and keep it readable!?

https://date-fns.org/v4.1.0/docs/isFuture